src/app/cube/home/splash/splash.component.ts

```diff
@@ -11,12 +11,12 @@ export class SplashComponent implements OnInit {
   numReleasedObjects = 0; // default number of released objects before the service provides a new number
 
   constructor(
-    private usageStatsService: UsageStatsService,
+    private metricService: MetricService,
     public googleTagService: GoogleTagService
     ) { }
 
   ngOnInit(): void {
-    this.usageStatsService.getLearningObjectStats().then(stats => {
+    this.metricService.getLearningObjectStats().then(stats => {
       this.numReleasedObjects = Math.floor(stats.released / 10) * 10;
```

Comment:
Just a question on this math. Why don't we just set numReleasedObjects = stats.released? Why do we divide by 10 and then multiply by 10?

Reply:
We do this so that we can display a round number in the home component. 

This is how its used:
![image](https://github.com/Cyber4All/clark-client/assets/72173743/e97b750b-c2cd-45af-9783-4f921e74a2e9)

For example if we have 1595 objects we divide by 10 to get 159.5, floor it to get 159 then multiply by 10 to get 1590. This way we can just add a `+` to the end of it.